storage/storage_test.go

```diff
@@ -178,9 +178,9 @@ func TestParse(t *testing.T) {
 func TestParseWithGraphDriverOptions(t *testing.T) {
 	optionLists := [][]string{
 		{},
-		{"unused1"},
-		{"unused1", "unused2"},
-		{"unused1", "unused2", "unused3"},
+		{"unused1=unused1"},
```

Comment:
CI now fails in `go test` with this error:
```
--- FAIL: TestParseWithGraphDriverOptions (0.00s)
    storage_test.go:95: vfs driver does not support unused1 options
```
Without this change, the test failed with a parse error.

I suspect this is due to changes in the updated containers/storage version I had to use to get cross-compilation to work, but I couldn’t find any related changes in that library. Any suggestions?

Reply:
My first instinct would be to pare this down; get either cross-compilation, or `go mod`, working _separately_. I don’t think they have to land at exactly the same time.

Sure, if we do have to upgrade, (and we do, fairly soon anyway), and if the upgrade breaks this code, this will have to be dealt with one way or another eventually.

Reply:
👍 Removed the containers/storage upgrade and the cross target from Travis CI and now it works (except for Travis CI flaking again).

Reply:
> My first instinct would be to pare this down; get either cross-compilation, or `go mod`, working _separately_. I don’t think they have to land at exactly the same time.

Ah, sorry, I missed that we already run `make cross` in the CI. We do want to keep that working, but at least now we are one step closer.

Reply:
Fixed cross build by updating containers/storage to v1.12.10.